sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdTokenTypeTests.java

```diff
@@ -0,0 +1,34 @@
+package com.azure.cosmos.implementation.directconnectivity.rntbd;
+
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class RntbdTokenTypeTests {
+
+    @Test(groups = { "unit" })
+    public void testAllTokenTypes() {
```

Comment:
we don't use "test" as prefix nor suffix as a test code style.
```suggestion
    public void allTokenTypes() {
```